pwndbg/wrappers/checksec.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import pwndbg.wrappers
+import subprocess
+
+
+@pwndbg.wrappers.OnlyWithCommand
+def get_raw_out():
```

Comment:
Just try it. Imho there shouldn't be a cyclic imports problem as `pwndbg.commands` is not using this module.